my_project/tab_sun/app_sun.py

```diff
@@ -131,17 +132,17 @@ def static_section():
     return html.Div(
         className="container-col full-width",
         children=[
-            html.Div(
-                children=title_with_tooltip(
-                    text="Global and Diffuse Horizontal Solar Radiation (Wh/m²)",
-                    tooltip_text=None,
-                    id_button="monthly-chart-label",
-                ),
-            ),
-            dcc.Loading(
-                type="circle",
-                children=html.Div(id="monthly-solar"),
-            ),
+            # html.Div(
```

Comment:
Why you have commented this code out?

Reply:
I am just reviewing the code from GitHub so I cannot verify this for you but I am going to close this issue if you are saying that the code is duplicated